packages/ketcher-react/src/script/editor/tool/reactionmap.ts

```diff
@@ -51,11 +51,7 @@ class ReactionMapTool {
       const ci = this.editor.findItem(event, ['atoms'], this.dragCtx.item)
       const atoms = rnd.ctab.molecule.atoms
 
-      if (
-        ci &&
-        ci.map === 'atoms' &&
-        isValidMap(this.rcs, this.dragCtx.item.id, ci.id)
-      ) {
+      if (ci && ci.map === 'atoms') {
```

Comment:
let's rename `ci` to `closestItem` shall we?
Developers that come after us will be very grateful